metagraph/src/graph/annotated_dbg.hpp

```diff
@@ -129,33 +127,33 @@ class AnnotatedDBG : public AnnotatedSequenceGraph {
                    size_t min_count = 0,
                    bool with_kmer_counts = false) const;
 
-    std::vector<std::pair<Label, std::vector<size_t>>>
+    std::vector<std::tuple<Label, size_t, std::vector<size_t>>>
```

Comment:
maybe add comments to describe the relationship between `size_t` and `std::vector<size_t>` represent